applications/calculator/main.cpp

```diff
@@ -25,148 +25,191 @@ enum CALCULATOR_OPERATOR
     OPERATOR_CLEAR_ALL = 6
 };
 
-void update_calculator_text()
+#define CALC_ID_BUTTON_0 "0"
```

Comment:
This is not what I meant sorry, widget IDs sould be "button0" instead of just "0" for exemple.